modules/core/04-channel/keeper/handshake_test.go

```diff
@@ -218,23 +165,17 @@ func (suite *KeeperTestSuite) TestChanOpenTry() {
 			path.EndpointB.UpdateConnection(func(c *connectiontypes.ConnectionEnd) {
 				c.Versions = append(c.Versions, connectiontypes.NewVersion("2", []string{"ORDER_ORDERED", "ORDER_UNORDERED"}))
 			})
-
-			suite.chainB.CreatePortCapability(suite.chainB.GetSimApp().ScopedIBCMockKeeper, ibctesting.MockPort)
-			portCap = suite.chainB.GetPortCapability(ibctesting.MockPort)
 		}, false},
 		{"connection does not support ORDERED channels", func() {
 			path.SetupConnections()
 			path.SetChannelOrdered()
 			err := path.EndpointA.ChanOpenInit()
 			suite.Require().NoError(err)
 
-			// modify connA versions to only support UNORDERED channels
-			path.EndpointA.UpdateConnection(func(c *connectiontypes.ConnectionEnd) {
+			// modify connB versions to only support UNORDERED channels
```

Comment:
This change was needed because the test was not correct. It comes from https://github.com/cosmos/ibc-go/pull/7009/commits/29d9b5966da6bfb41a19e8fe74aec3c92a0ff87a (comments on #7009 explain it better)